### PR TITLE
Add tests to localrpc package

### PR DESCRIPTION
**Pull request by gusjavaz**
## Scope
Unit tests for localrpc package with 65% Coverage 
Tests start an ethereumless validator node in 8889 with a minimal DB for handler testing
A bug was detected and fixed on validator.go. The error prevented the initialization of gossip handlers with chainIds other than 0

## Todos
Add tests for mined transactions, currently only pending are tested

### CODE REVIEW

**Comment by vtleonardo** [comment]
Please, merge the latest candidate branch on this PR and check if CI is going to pass. In case you find any problem with the CI, let me know!

**Review by vtleonardo — COMMENTED** [review]
Do we need to add the package-lock.json? There's any new node package being added? In case yes, you need to commit the package.json as well in order to make the CI pass.

File: localrpc/setup_test.go [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+	go consDlManager.Start()
+	defer consDlManager.Close()
+
+	go localStateHandler.Start()
+	defer localStateHandler.Stop()
+
+	go consGossipHandlers.Start()
+	defer consGossipHandlers.Close()
+
+	go storage.Start()
```
Comment (vtleonardo) [inline localrpc/setup_test.go:134]: don't we need a `defer storage.Close()` here?

Comment (gusjavaz) [reply localrpc/setup_test.go:134]: Agree but actually there is not Close() method on Storage (or anything similar)

**Comment by gusjavaz** [comment]
> Do we need to add the package-lock.json? There's any new node package being added? In case yes, you need to commit the package.json as well in order to make the CI pass.

You are right, there is no need. Checked out file from candidate as you mentioned.

**Pull request merged**